js/side-panel.js

```diff
@@ -0,0 +1,87 @@
+var PanelManager = {
+  TRANSITION_TIME: 300,
+  TOGGLE_DISTANCE: 218,
+
+  init: function(e) {
+    $('.side-panel').load('templates/side-panel.html', function(e) {
+      $('#btnToggle').on('click', function(e) {
+        $('#slider').animate({
+          'right' : $('#slider').css('right') == '0px' ? - PanelManager.TOGGLE_DISTANCE + 'px': '0px'
```

Comment:
There will render error "Uncaught ReferenceError: right is not defined" when the single quotes are removed.